ElectricityService.App/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
File should not exist in the repository, can be fixed by updating the gitignore

Reply:
did so, @Aplicticus can verify Still VS insists that the file should be committed...